Assets/Scripts/Characters/ArtificialInput.cs

```diff
@@ -55,7 +55,7 @@ public class ArtificialInput : ComponentFeature<Character> {
                     closestPrey = hit.transform;
                 }
                 float hitDistance = Mathf.Abs(Vector3.Distance(transform.position, hit.transform.position));
-                float closestPreyDistance = Mathf.Abs(Vector3.Distance(transform.position, closestPrey.position));
+                float closestPreyDistance = Mathf.Abs(Vector3.Distance(character.attackPoint.position, closestPrey.position));
```

Comment:
Vector3.Distance ist immer positiv.
```suggestion
                float closestPreyDistance = Vector3.Distance(character.attackPoint.position, closestPrey.position);
```